devops/azure-pipelines/discord-job-template.yml

```diff
@@ -1,25 +1,30 @@
+parameters:
+  - name: 'dependOn'
+    type: string
```

Comment:
and [here](https://github.com/1058-Sophia-Gavrila-Forge/Forge-Backend/pull/2#pullrequestreview-710831944)

Reply:
not sure what is meant by this, if there is still a problem after the changes made, please explain the issue further.

Reply:
still a problem. the indentation of sequences is inconsistent in your yaml files. the `- ` syntax indicates sequences. yaml allows you to either keep the `-` even with the line above the first one, or, indent it an extra level. just be consistent.